src/kOS.Safe.Test/Serialization/TerminalFormatterTest.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using kOS.Safe.Encapsulation;
+using kOS.Safe.Exceptions;
+using NUnit.Framework;
+using kOS.Safe.Serialization;
+
+namespace kOS.Safe.Test.Serialization
+{
+    [TestFixture]
+    public class TerminalFormatterTest
+    {
+        [SetUp]
+        public void Setup()
+        {
+
+        }
+
+        [Test]
+        public void CanSerializeLexicons()
+        {
+            var lex = new Lexicon();
+            var nested = new Lexicon();
+
+            lex["key1"] = "value1";
+            lex["key2"] = 1;
+            lex["key3"] = nested;
+
+            nested["nested1"] = "nested1value";
+            nested["nested2"] = "nested2value";
+
+            Assert.AreEqual("LEXICON of 3 items:\n[\"key1\"]= value1\n[\"key2\"]= 1\n[\"key3\"]=" +
+                " LEXICON of 2 items:\n  [\"nested1\"]= nested1value\n  [\"nested2\"]= nested2value", Serialize(lex));
```

Comment:
Hang on a second here.  Where is the standard that says we'll use Environment.Newline everywhere?  Because we certainly don't.  I know many places that hardcode '\n' in our code,  One of them is the PRINT statement printing to the terminal - it assumes that if it sees a '\n' you will get a \r\n effect even if the \r is missing.